docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,107 @@
+- ***new!*** [Ubuntu 16.04 Package Manager - Install .NET Core](../core/install/linux-package-manager-ubuntu-1604.md) - Design new .NET Core Setup/Install area; Rewrite prereqs articles; Add package managers
+- ***new!*** [Ubuntu 18.04 Package Manager - Install .NET Core](../core/install/linux-package-manager-ubuntu-1804.md) - Design new .NET Core Setup/Install area; Rewrite prereqs articles; Add package managers
+- ***new!*** [Ubuntu 19.04 Package Manager - Install .NET Core](../core/install/linux-package-manager-ubuntu-1904.md) - Design new .NET Core Setup/Install area; Rewrite prereqs articles; Add package managers
+- ***new!*** [Install the .NET Core SDK](../core/install/sdk.md) - Design new .NET Core Setup/Install area; Rewrite prereqs articles; Add package managers
+- ***new!*** [Run-time configuration options for garbage collection](../core/run-time-config/garbage-collector.md) - Add run-time GC settings
+- [dotnet sln](../core/tools/dotnet-sln.md) - Update dotnet sln for 3.0
+- [What's new in .NET Core 3.0](../core/whats-new/dotnet-core-3-0.md) - Rearrange into categories
+
+## C# language
+
+- ***new!*** [C# docs - get started, tutorials, reference.](../csharp/index.yml) - New csharp landing page
+- ***new!*** [Nullable value types (C# reference)](../csharp/language-reference/builtin-types/nullable-value-types.md) - Consolidated the C# nullable value types articles
+
+## .NET Core desktop
+
+- ***new!*** [Create a template for a control](../desktop-wpf/themes/how-to-create-apply-template.md) - Add desktop wpf article for templates
+
+## .NET framework
+
+- [Interop Marshaling](../framework/interop/interop-marshaling.md) - Update interop-marshaling.md
+- [Using TCP Services](../framework/network-programming/using-tcp-services.md) - Modernizing code and remove extra spaces
+- [Application Domain Resource Monitoring (ARM) ETW Events](../framework/performance/application-domain-resource-monitoring-arm-etw-events.md) - Update application-domain-resource-monitoring-arm-etw-events.md
+- [Garbage Collection ETW Events](../framework/performance/garbage-collection-etw-events.md) - Update garbage-collection-etw-events.md
+- [.NET Framework Tools](../framework/tools/index.md) - Remove duplicated sentence
+- [How to create and run a long-running workflow](../framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md) - Update how-to-create-and-run-a-long-running-workflow.md
+- [Threading Model](../framework/wpf/advanced/threading-model.md) - Fix up xref.xref instances
+- [Data Templating Overview](../framework/wpf/data/data-templating-overview.md) - Fix up xref.xref instances
+
+## F# language
+
+- ***new!*** [What's new in F# 4.5](../fsharp/whats-new/fsharp-45.md) - What's new with F#?
+
+## Community contributors
+
+The following people have contributed to .NET docs in the last month. Thank you!
```

Comment:
Can we add a link on how to contribute to docs? 

https://docs.microsoft.com/contribute/ would sound about right.

Reply:
I added that link to the landing page, and pointed to the landing page on each sprint what's new page.